ASPNetCoreMasters/Controllers/ItemsController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using ASPNetCoreMasters.BindingModels;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Services;
+using Services.DTO;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ASPNetCoreMasters.Controllers
+{
+    public class ItemsController : ControllerBase
+    {   
+        public int Get(int id)
+        {
+            //var item = new ItemService();
+            //var result = item.GetAll(id);
+
+            return id;
+        }
+
+        public IActionResult Post(ItemCreateBindingModel itemCreateBidingModel)
```

Comment:
Please use [FromBody] tag so that model binding can occur. Also, please check the model state before performing the business logic in the action method. Model validation occurs automatically but we need to manually check the result